testbed/tests/results/TESTRESULTS.md

```diff
@@ -1,14 +1,18 @@
 # Test Results
-Started: Tue, 25 Jun 2019 10:31:56 -0400
+Started: Mon, 01 Jul 2019 23:04:29 +0530
 
-Test|Result|Duration|CPU Avg%|CPU Max%|RAM Avg MiB|RAM Max MiB|Sent Spans|Received Spans
-----|------|-------:|-------:|-------:|----------:|----------:|---------:|-------------:
-TestIdleMode|Passed|10s|0.3|1.0|25|34|0|0
-Test10kSPS|Passed|17s|91.4|95.7|35|43|149610|149610
-TestNoBackend10kSPS|Passed|10s|59.1|59.3|29|40|99910|0
-Test1000SPSWithAttributes/0*0bytes|Passed|12s|18.7|19.3|30|41|10000|10000
-Test1000SPSWithAttributes/100*50bytes|Passed|12s|70.2|71.0|32|45|10000|10000
-Test1000SPSWithAttributes/10*1000bytes|Passed|12s|68.0|74.3|32|44|10000|10000
-Test1000SPSWithAttributes/20*5000bytes|Passed|12s|158.2|173.7|39|55|10000|10000
+Test                                    |Result|Duration|CPU Avg%|CPU Max%|RAM Avg MiB|RAM Max MiB|Sent Spans|Received Spans
+----------------------------------------|------|-------:|-------:|-------:|----------:|----------:|---------:|-------------:
+TestIdleMode                            |PASS  |     10s|     0.3|     0.7|         24|         32|         0|             0
+Test10kSPS                              |PASS  |     17s|    73.3|    78.3|         34|         42|    148760|        148760
+TestNoBackend10kSPS                     |PASS  |     10s|    79.5|    81.7|         29|         40|     98770|             0
+Test1000SPSWithAttributes/0*0bytes      |PASS  |     12s|    25.8|    26.7|         30|         41|     10000|         10000
+Test1000SPSWithAttributes/100*50bytes   |PASS  |     12s|    87.1|    93.7|         34|         47|      9990|          9990
+Test1000SPSWithAttributes/10*1000bytes  |PASS  |     12s|    67.8|    74.3|         32|         44|     10000|         10000
+Test1000SPSWithAttributes/20*5000bytes  |PASS  |     12s|    96.7|   102.0|         46|         71|     10000|         10000
+TestBallast1000SPSWithAttributes/0*0bytes|FAIL  |      4s|    25.3|    25.3|         41|         82|      2830|          2790
```

Comment:
Was this supposed to fail?

Reply:
No. Let's merge #85 first and then I'll rebase this which'll fix it.

Reply:
Fixed.